instruqt-tracks/terraform-cloud-aws-v2/track_scripts/setup-workstation

```diff
@@ -0,0 +1,191 @@
+#!/bin/bash -l
+# Track setup script
+
+set -e
+
+# Wait for Instruqt to finish booting the VM
+# This is better than doing sleep
+while [ ! -f /opt/instruqt/bootstrap/host-bootstrap-completed ]
+do
+    echo "Waiting for Instruqt to finish booting the VM"
+    sleep 1
+done
+
+# To repurpose this script, we choose the target 
+# Cloud Environment using this variable
+# export CLOUD_ENV="aws"
+
+# We use GitHub gh to load secrets in our working repo
+# For everything else, we use git
+snap install gh
+
+# Ensure we're working on the home directory
+cd /root
+
+# Get our working assets
+echo "Cloning assets..."
+git clone https://github.com/hashicorp/se-workshop-hashicat-app.git
+echo "Done cloning assets..."
+
+mv /root/se-workshop-hashicat-app/hashicat-$CLOUD_ENV /root/hashicat-$CLOUD_ENV
+mv /root/se-workshop-hashicat-app/hashicat-policies /root/hashicat-$CLOUD_ENV/policies
```

Comment:
This through line 36 took me a bit to figure out, doesn't it make sense to do
```
mkdir /root/hashicat-$CLOUD_ENV/policies
mv /root/se-workshop-hashicat-app/hashicat-policies/$CLOUD_ENV/* /root/hashicat-$CLOUD_ENV/policies
```

We get rid of lines 34-36, which assume `$CLOUD_ENV` is `aws`, and we're not copying stuff just to move a subset of it up a directory level and delete the rest. Seems clearer intention to me.

Reply:
This is removing the original - which we avoided in later revisions.

mv /root/se-workshop-hashicat-app/hashicat-$CLOUD_ENV /root/hashicat-$CLOUD_ENV
mv /root/se-workshop-hashicat-app/hashicat-policies/$CLOUD_ENV /root/hashicat-$CLOUD_ENV/policies